DNN Platform/DotNetNuke.Abstractions/Application/SchedulerMode.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Abstractions.Application;
+
+/// <summary>The method for triggering scheduled tasks.</summary>
+public enum SchedulerMode
+{
+    /// <summary>The scheduler is disabled.</summary>
+    Disabled = 0,

```

Comment:
`TIMER_METHOD` and `REQUEST_METHOD` are added here, should `DISABLED` also be added?

Reply:
Never mind, I forgot that this isn't possible due VB.NET being case insensitive. 😅

Reply:
Yeah, I added it in all caps to start and that triggered a warning about case sensitivity.